ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -738,6 +738,7 @@ literals:
 - position_status
 - condensing_water_valve_percentage_sensor
 - entering_cooling_coil_temperature_sensor
+- fire_fault_status
```

Comment:
how is this different than fire alarm?

Reply:
Description updated. More sub-details about this fire_fault_status.